catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/LatestCommand.java

```diff
@@ -66,11 +66,11 @@ protected Object executeWithSubject() throws Exception {
     console.printf(formatString, "", "", "", "");
     printHeaderMessage(String.format(formatString, NUMBER, ID, DATE, TITLE));
 
-    Filter filter = filterBuilder.attribute(Metacard.MODIFIED).before().date(new Date());
+    Filter filter = filterBuilder.attribute(Core.METACARD_MODIFIED).before().date(new Date());
```

Comment:
This line needs to be updated too: https://github.com/codice/ddf/blob/080dea730aa7c06240cb97e74cc5b43203242242/distribution/docs/src/main/resources/content/_running/catalog-commands.adoc#L44

Reply:
Oh nice catch!!